sequencer/src/state.rs

```diff
@@ -178,7 +197,7 @@ pub struct FeeAmount(U256);
 #[derive(
     Default, Hash, Copy, Clone, Debug, Deserialize, Serialize, PartialEq, Eq, PartialOrd, Ord,
 )]
-pub struct FeeAccount(Address);
+pub struct FeeAccount(pub Address);
```

Comment:
Nit: could also derive `from`, `into` instead of exposing this field

Reply:
fixed in f5e45de1b12025ecaf5ae4eab0f7a90c90c6c29e